geometry_msgs/Pose detection
---
geometry_msgs/Pose approaching_point